### PR TITLE
Add a banner and button to the welcome screen introducing the Stately visual editor

### PR DESCRIPTION
**Pull request by laurakalbag**
Before:

CleanShot 2023-10-31 at 17 28 54@2x

After:

CleanShot 2023-11-03 at 11 01 39@2x

### CODE REVIEW

**Comment by farskid** [comment]
@laurakalbag @davidkpiano I'd vote for a much stronger tone indicating that viz is already deprecated and no longer actively maintained and if they want their visualizations to continue to work, they'd need to migrate to Studio asap. The current banner comes off more as an advertisement for a complimentary product.

**Comment by laurakalbag** [comment]
> @laurakalbag @davidkpiano I'd vote for a much stronger tone indicating that viz is already deprecated and no longer actively maintained and if they want their visualizations to continue to work, they'd need to migrate to Studio asap. The current banner comes off more as an advertisement for a complimentary product.

@farskid any specific suggestions for wording?

**Comment by laurakalbag** [comment]
CleanShot 2023-11-01 at 13 56 06@2x

**Review by davidkpiano — APPROVED** [review]

File: src/WelcomeArea.tsx [hunk 1]
```diff
@@ -129,45 +130,109 @@ export const WelcomeArea = () => {
...
+                </Text>
+                <Stack backgroundColor="gray.700" spacing="5" padding="6">
+                  <Text>
+                    This legacy visualizer is deprecated and is no longer
+                    maintained.
+                  </Text>
+                  <Text>
+                    For the best experience,{' '}
+                    <Link
+                      href="https://stately.ai?source=viz"
```
Comment (farskid) [inline src/WelcomeArea.tsx:149]: Why not linking to the editor directly?
`https://stately.ai/editor?source=viz`

File: src/WelcomeArea.tsx [hunk 2]
```diff
@@ -129,45 +130,109 @@ export const WelcomeArea = () => {
...
+                      {' '}
+                      Read our docs for more information on the differences
+                      between the legacy visualizer and new visualizer.
+                    </Link>
+                  </Text>
+                  <Text>
+                    <Button
+                      {...buttonStyleProps}
+                      as={Link}
+                      href="https://stately.ai?source=viz"
```
Comment (farskid) [inline src/WelcomeArea.tsx:177]: Same here.

**Review by davidkpiano — APPROVED** [review]

File: src/WelcomeArea.tsx (hunk 1 above)
Comment (laurakalbag) [reply src/WelcomeArea.tsx]: Good catch, thank you! Fixed.

File: src/WelcomeArea.tsx (hunk 2 above)
Comment (laurakalbag) [reply src/WelcomeArea.tsx]: Fixed 😊

**Pull request merged**